### PR TITLE
Setup waitlist frame

### PR DESCRIPTION
**Pull request by motechFR**

### CODE REVIEW

File: apps/connect/app/waitlist/[fid]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { baseUrl } from '@root/config/constants';
+
+import WaitlistGif from 'public/images/waitlist/waitlist-intro.gif';
```
Comment (mattcasey) [inline apps/connect/app/waitlist/[fid]/page.tsx:3]: remember this should be hosted inside apps/farcaster, not connect

**Review by mattcasey — CHANGES_REQUESTED** [review]
All Farcaster-related features should live in apps/farcaster now

**Review by mattcasey — APPROVED** [review]

File: apps/farcaster/app/api/waitlist/route.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    return new Response(html, {
+      status: 200,
+      headers: {
+        'Content-Type': 'text/html'
+      }
+    });
+  }
+
+  prettyPrint({ validatedMessage });
```
Comment (valentinludu) [inline apps/farcaster/app/api/waitlist/route.tsx:64]: I think you can delete this one and add a log.info.

File: apps/farcaster/app/waitlist/[fid]/page.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { baseUrl } from '@root/config/constants';
+import { headers } from 'next/headers';
+
+import { JoinWaitlistHome } from 'components/waitlist/joinWaitlist/JoinWaitlistHome';
+import WaitlistGif from 'public/images/waitlist/waitlist-intro.gif';
+
+// async function inboundReq(input: any) {
```
Comment (valentinludu) [inline apps/farcaster/app/waitlist/[fid]/page.tsx:7]: You can delete comments if not needed.

**Pull request merged**